src/kasse.c

```diff
@@ -269,21 +269,24 @@ void buy_custom(void) {
 }
 
 void set_time_interactive(void) {
-  BYTE part[3] = {'0', '0', '\0'};
-  BYTE tp1, tp2, tp3;
+  char part[3] = {'\0', '\0', '\0'};
+  uint8_t day, tp1, tp2, tp3;
   char *time_input, *time;
-  cprintf("Gib die aktuelle Uhrzeit ein (Format HHMMSS):\r\n");
+  cprintf("Gib den aktuellen Tag des Events und Uhrzeit ein\r\n"
+          "(Format DHHMMSS):\r\n");
```

Comment:
One character for the day? Does the day start at 0 or 1? Can we add an example value?

Reply:
yeah, I haven't thought about this yet. What do you propose?

Reply:
I’d suggest to start at day 0. How about:
```
(Format DHHMMSS, 0-indexiert, z.B. 0174259):\r\n
```